Firmware/Tcodes.cpp

```diff
@@ -57,14 +57,14 @@ void TCodes(char *const strchr_pointer, uint8_t codeValue) {
         }
     } else {
         SChooseFromMenu selectedSlot;
-//        if (strchr_pointer[index] == '?')
-//            selectedSlot = TCodeChooseFromMenu();
-//        else {
+        if (strchr_pointer[index] == '?') {
+            selectedSlot = TCodeChooseFromMenu();
+        /*} else if (MMU2::mmu2.Enabled() && SpoolJoin::spooljoin.isSpoolJoinEnabled()) {
+            // TODO: What if the next slot has no filament?
```

Comment:
I think the original idea was to select next slot again of there was no filament, but we can skip that for now.